vue-example-plugin/package.json

```diff
@@ -14,7 +14,6 @@
     "@penpot/plugin-styles": "^0.8.0",
     "@penpot/plugin-types": "^1.0.0",
     "vue": "^3.4.21",
```

Comment:
please remove the extra comma in the vue dependency so it doesn't fail when installing

Reply:
Hello, it's done.